mm2src/mm2_main/tests/mm2_tests/lightning_tests.rs

```diff
@@ -207,15 +208,48 @@ fn test_enable_lightning() {
 
     let _electrum = block_on(enable_electrum(&mm, "tBTC-TEST-segwit", false, T_BTC_ELECTRUMS));
 
-    let enable_lightning = block_on(enable_lightning(&mm, "tBTC-TEST-lightning", 600));
-    assert_eq!(&enable_lightning.platform_coin, "tBTC-TEST-segwit");
+    let enable_lightning_coin = block_on(enable_lightning(&mm, "tBTC-TEST-lightning", 600));
+    assert_eq!(&enable_lightning_coin.platform_coin, "tBTC-TEST-segwit");
     assert_eq!(
-        &enable_lightning.address,
+        &enable_lightning_coin.address,
         "02ce55b18d617bf4ac27b0f045301a0bb4e71669ae45cb5f2529f2f217520ffca1"
     );
-    assert_eq!(enable_lightning.balance.spendable, BigDecimal::from(0));
-    assert_eq!(enable_lightning.balance.unspendable, BigDecimal::from(0));
+    assert_eq!(enable_lightning_coin.balance.spendable, BigDecimal::from(0));
+    assert_eq!(enable_lightning_coin.balance.unspendable, BigDecimal::from(0));
+    // Disable tBTC-TEST-segwit
```

Comment:
Please add a space here. Also, please first disable `tBTC-TEST-lightning` then re-enable it, before disabling `tBTC-TEST-segwit` and re-enabling both coins.
We need to make sure the futures are stopped in both cases since before this PR we couldn't re-enable lightning after disabling it due to the port used for listening to incoming connections not being released, as it was not dropped by a thread that was not aborted when disabling the coin. It was released when using the `stop` RPC, so it might be dropped when disabling the platform coin but not dropped when disabling lightning only. That's why we need to cover both cases to not regress on this.

Reply:
understand that..thanks